### ISSUE TITLE
Error adding note to manually created item

### ISSUE THREAD

**Issue by hloliver** [PTFS-Europe/rebus-list-2#183]
Hi

I manually created an item in this list: https://tavi.rebuslist.co.uk/#/list/1994

I then went to Search so that I could edit this record and add an entry in the notes field, but got the following error:

**There is a problem with your form data, please check it and try again
This message will disappear when your form data is valid**

and this message was underneath the record: **Invalid type, expected array**

![error adding note to manually created item](https://cloud.githubusercontent.com/assets/12201255/17393250/6d9a20bc-5a1a-11e6-9b61-4f64f08ffff1.png)

Can you take a look?
Thanks
Helen

**Comment by hloliver** [comment]
An update on this issue - today I tried to manually add an article to this list https://tavi.rebuslist.co.uk/#/list/1997 - when I got to the notes field a similar error occurred, I wasn't able to complete the record without removing the note.  I clicked on Finish and was taken back to the search results screen, I clicked on Finish again and got the red error flags at the bottom of the screen.

The article that I added was displayed in the list but when I published the list it disappeared!

I managed to get most of this on video: http://screencast.com/t/vrHcZdpf1SD

When I tried a similar workflow earlier I saw the same errors, but the record wasn't saved at all.

best wishes
Helen
